x-pack/plugins/lens/public/pie_visualization/render_function.tsx

```diff
@@ -82,6 +82,7 @@ export function PieComponent(
     legendPosition,
     nestedLegend,
     percentDecimals,
+    donutInnerAreaSize,
```

Comment:
I don't understand why you use  `donutInnerAreaSize` name instead of generic `emptySizeRatio`?